data/projects.json

```diff
@@ -25,5 +25,14 @@
         "category": "Website",
         "site": "https://roshaen.github.io/currency",
         "tags": "html, css, js"
+    },
+    {
+        "name": "Paint-with-UG",
+        "display": "Paint with UG",
+        "description": "This is a Desktop Application in which you can create shape,fill color,change background etc.",
+        "author": "Ujjwal",
```

Comment:
Hi @UG-SEP,
This is not your github username

Reply:
ok